src/Autofac/Features/Collections/CollectionRegistrationSource.cs

```diff
@@ -62,7 +63,7 @@ public IEnumerable<IComponentRegistration> RegistrationsFor(Service service, Fun
 
         if (registrationAccessor == null)
         {
-            throw new ArgumentNullException(nameof(registrationAccessor));
+            throw new ArgumentNullException(nameof(registrationAccessor));`
```

Comment:
```suggestion
            throw new ArgumentNullException(nameof(registrationAccessor));
```

I think the extra backtick here is hosing you up.

Reply:
🤦‍♂️